homeassistant/components/xiaomi_miio/const.py

```diff
@@ -286,7 +279,7 @@
 FEATURE_FLAGS_AIRFRESH = (
     FEATURE_SET_BUZZER
     | FEATURE_SET_CHILD_LOCK
-    | FEATURE_SET_LED
+    | FEATURE_SET_LED_BRIGHTNESS
```

Comment:
Is this related to the switch platform? FEATURE_SET_LED_BRIGHTNESS is used in the select platform.

Reply:
iirc python-miio has both bool & preset type of settings for led brightness, maybe the switch one was for on/off instead of predefined presets?

Reply:
I think that for one device the led switch and the brightnes led selector (with the off option) is too much. That's why I deleted the feautre set led.

Reply:
This will be a regression for those users who want to toggle the state, so I'm not sure if this is a good idea? I'll try to keep that in my mind when planning the common interface, maybe that should be a preseted (on/off being the options for the boolean toggle).

Reply:
At the common interface we should merge this into one setter. At the moment it's not consistent.

Reply:
Yes, it's going to be kind of regression, so I'm reverting this change.